src/server/datastore/index.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+{"users":[{"id":1,"firstname":"Saheed","email":"Shittu","lastname":"wale","othername":"saheed@gmail.com","phoneNumber":"08099999","password":"12345","isAdmin":true},{"id":2,"firstname":"Kola","email":"kunle","lastname":"tunde","othername":"quduskunle@gmai.com","phoneNumber":"08052327990","password":"123ewr","isAdmin":true},{"id":3,"firstname":"Saheed123333","email":"saheed@gmail.com","lastname":"Adewale","othername":"wale","phoneNumber":"08023454546","passportUrl":"https://www.google.com","password":"1234567","isAdmin":false},{"id":4,"firstname":"Saheed123333","email":"saheed@gmail.com","lastname":"Adewale","othername":"wale","phoneNumber":"08023454546","passportUrl":"https://www.google.com","password":"1234567","isAdmin":false},{"id":5,"firstname":"Saheed123333","email":"saheed@gmail.com","lastname":"Adewale","othername":"wale","phoneNumber":"08023454546","passportUrl":"https://www.google.com","password":"1234567","isAdmin":false},{"id":6,"firstname":"Saheed123333","email":"saheed@gmail.com","lastname":"Adewale","othername":"wale","phoneNumber":"08023454546","passportUrl":"https://www.google.com","password":"1234567","isAdmin":false},{"id":7,"firstname":"Saheed123333","email":"saheed@gmail.com","lastname":"Adewale","othername":"wale","phoneNumber":"08023454546","passportUrl":"https://www.google.com","password":"1234567","isAdmin":false},{"id":8,"firstname":"Saheed","email":"walewale@gmail.com","lastname":"Shittu","othername":"Adewale","phoneNumber":"08065121234","passportUrl":"https://passorturl.io","password":"041419122","isAdmin":false},{"id":9,"firstname":"Saheed","email":"walewale@gmail.com","lastname":"Shittu","othername":"Adewale","phoneNumber":"08065121234","passportUrl":"https://passorturl.io","password":"041419122","isAdmin":false}],"party":[{"id":1,"userId":3,"name":"apc","hqAddress":"Ilupeju, Lagos","logoUrl":"image logo"},{"id":2,"name":"Adewale","hqAddress":"12, Obanikoro Lagos, Nigeria","logoUrl":"https://www.adewale.com"},{"id":3,"name":"Adewale","hqAddress":"12, Obanikoro Lagos, Nigeria","logoUrl":"https://www.adewale.com"},{"id":4,"name":"Adewale","hqAddress":"12, Obanikoro Lagos, Nigeria","logoUrl":"https://www.adewale.com"}],"votes":[],"offices":[]}
```

Comment:
Rename the json file to javascript file and use javascript data structure to write or read file